docs/DeveloperGuide.md

```diff
@@ -170,6 +170,69 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Click Intern Card To View Details
```

Comment:
Should it be 'Intern Card' or 'Internship Application Card'?
Can also be just 'Click Internship Application Card' instead of the additional 'to View Details'

Reply:
Oh! Good catch, you are right.